taskcluster/ci/ui-test/kind.yml

```diff
@@ -80,3 +80,16 @@ jobs:
                 - [automation/taskcluster/androidTest/ui-test.sh, x86-beta-tests, app.apk, android-test.apk, '50']
         treeherder:
             symbol: beta(ui-test-x86-beta)
+    x86-nightly:
+        attributes:
+            build-type: nightly-firebase
+        description: Test Fenix
+        run-on-tasks-for: [github-push]
+        dependencies:
+            signing: signing-nightly-firebase
+            signing-android-test: signing-android-test-nightly
+        run:
+            commands:
+                - [automation/taskcluster/androidTest/ui-test.sh, x86-start-nightly-test, app.apk, android-test.apk, '50']
```

Comment:
do we want 50 here?

Reply:
I don't think so 😓 It would be enough with one since we are running only one test atm. According to the [comments](https://github.com/mozilla-mobile/fenix/blob/master/automation/taskcluster/androidTest/ui-test.sh#L27), that could be -1:
```
echo "To run UI tests on ARM device shard (1 test / shard)"
echo "$ ui-test.sh arm64-v8a -1"
```